lib/instances/UIListLayout_spec.lua

```diff
@@ -1,16 +1,22 @@
-local Instance = import("../Instance")
-local typeof = import("../functions/typeof")
+local UIListLayout = import("./UIListLayout")
+local UDim = import("../types/UDim")
 
 describe("instances.UIListLayout", function()
 	it("should instantiate", function()
-		local instance = Instance.new("UIListLayout")
+		local instance = UIListLayout:new()
 
 		assert.not_nil(instance)
 	end)
 
+	it("should inherit from UIGridStyleLayout", function()
+		local instance = UIListLayout:new()
+
+		assert.True(instance:IsA("UIGridStyleLayout"))
+	end)
+
 	it("should have properties defined", function()
-		local instance = Instance.new("UIListLayout")
-		assert(typeof(instance.Padding), "UDim")
-		assert(typeof(instance.SortOrder), 0)
+		local instance = UIListLayout:new()
+
+		assert.equals(getmetatable(instance.Padding).type, UDim)
```

Comment:
I think we can use `typeof(instance.Padding) == "UDim")` here, since this right now is just doing what `typeof` does.